lib/zclient.h

```diff
@@ -537,6 +555,16 @@ struct zapi_route {
  * offload situation.
  */
 #define ZEBRA_FLAG_OFFLOAD_FAILED     0x200
+/*
+ * This flag tells Zebra that the route is a seg6 route and should
+ * be treated specially.
+ */
+#define ZEBRA_FLAG_SEG6_ROUTE         0x400
```

Comment:
also, note the comment above about `zclient_dump_route_flags()` - if we decide to keep these in this place, you'll need to update that function too.

Reply:
As per my answer above, this define has been removed.

ref: https://github.com/FRRouting/frr/pull/5865#discussion_r619194849